be/src/storage/utils.cpp

```diff
@@ -390,4 +391,9 @@ std::string file_name(const std::string& fullpath) {
     return path.filename().string();
 }
 
+std::string _mask_token(const std::string& str) {
+    std::regex pattern("token=[\\w|-]+");
```

Comment:
looks like this is just for the specific case where it is known that there is a `token` query parameter and it is sensitive information.

A more generic way may be: mask the `user:password` in the URL authority part as well if exists. and mask out all the values of the query parameters.

Reply:
@kevincai yes.. i did consider the option to mask all the values of the query parameters; but then if we do that then it would not provide any useful information to debug the issue. so i did not opt for this solution. what is your opinion?

Reply:
It's a tradeoff between debug-friendly and security.

what I can think of is, adding a configvar to turn on/off the masking, so when need to debug the issue, turn off the masking, and by default the masking is turned on so the info can be secured.

Reply:
fixed